### ISSUE TITLE
Non-carto visualizers

### ISSUE THREAD

**Issue by jgoizueta** [CartoDB/franchise#26]
I hadn't used this for a while and it's still awesome, and it can now use cartojs/cartovl, wow!

But I found something missing, I wanted to see a tabular view of my query and there isn't an option for that anymore. It seems you removed it in https://github.com/CartoDB/franchise/commit/02fc82e5619d4179169afd8d70d4c158e480585d @alrocar 😠

Cannot those options co-exist with the new cartojs/cartovl views? I found really useful to be able to switch to a table or graph view of the query. 🙏

**Comment by alrocar** [comment]
yep, the problem was the the current widgets, loaded everything into memory, so whenever you had a "big" dataset, the application crashed. So I decided to remove them temporarily.

I'll deploy `master` (which contains the widgets) as a separate application so we can access both from different URLs.

cc. @arredond which complained about the same stuff xD

I'll keep you both posted 🙏

**Comment by arredond** [comment]
Sounds awesome. Thanks @alrocar!!

The folks over at Geographica will be eager to use it too :)

**Comment by alrocar** [comment]
https://franchise-master.carto.io now points to master (it includes all the widgets but not CARTO VL).

**Comment by arredond** [comment]
Awesome @alrocar, thanks!

Do you think there's any prospect of merging these two? I haven't really understood why CARTO VL is incompatible with big datasets but `carto.js` is. Don't they both bring all data into memory? I actually thought it was the opposite: carto.js brings in all the data but CARTO VL just brings in the tiles within the viewport. Please correct me if I'm wrong (which I surely am :D )

**Comment by alrocar** [comment]
No plans at the moment.

The problem was I was using CARTO VL for bigger datasets, since it makes use of "smart aggregations"™️ it works fine, but the rest of the widgets, still needed the raw data to work locally and they are not designed for that.

Neither CARTO VL nor carto.js need the raw data locally to work, that's why I just removed the widgets... that way I still was able to make maps without any dataset size limitation.

One nice solution would be to use smarter widgets, that make use of aggregations as well :)
